src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4804,7 +4804,8 @@ void SizeSysOutdoorAir(EnergyPlusData &state)
                                     .TotalOAFromArea; // should not have diversity at this point (no should have diversity in Vou if VRP)
                     if (SysSizInput(SysSizNum).SystemOAMethod == SOAM_ZoneSum) { // ZoneSum Method
                         SysOAUnc += ZoneOAUnc;
-                    } else if (SysSizInput(SysSizNum).SystemOAMethod == SOAM_VRP) { // Ventilation Rate Procedure
+                    } else if (SysSizInput(SysSizNum).SystemOAMethod == SOAM_VRP ||
+                               SysSizInput(SysSizNum).SystemOAMethod == SOAM_SP) { // Ventilation Rate Procedure
```

Comment:
Most of the VRP calculation also apply to the SP, the only difference is the calculation of the ventilation efficiency (Ev) and the zone minimum primary airflow (Vpz_min).